docs/development/application-registry.md

```diff
@@ -41,6 +41,10 @@ A dictionary of particular features (e.g. custom fields) mapped to the NetBox mo
 
 Supported model features are listed in the [features matrix](./models.md#features-matrix).
 
+### `models`
```

Comment:
I think this is a bit confusing and should be stated the opposite - you need to set _netbox_private to True to make it non public.